### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by shishurajpandey**
Summary-
Just a minor code improvement, doesn't affect functionality.

### Checklist

- [x ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: java/src/com/thoughtworks/selenium/Selenium.java [hunk 1]
```diff
@@ -1705,7 +1705,7 @@ public interface Selenium {
 
   /**
    * Tells the Selenium server to add the specified key and value as a custom outgoing request
-   * header. This only works if the browser is configured to use the built in Selenium proxy.
+   * header. This only works if the browser is configured to use the builtin Selenium proxy.
```
Comment (titusfortner) [inline java/src/com/thoughtworks/selenium/Selenium.java]: built-in

**Review by titusfortner — CHANGES_REQUESTED** [review]
Thanks for the submission! "behavior" is correct for US English; "behaviour" is British English.

File: java/src/com/thoughtworks/selenium/Selenium.java (hunk 1 above)
Comment (shishurajpandey) [reply java/src/com/thoughtworks/selenium/Selenium.java]: Resolved.

**Comment by titusfortner** [comment]
Ran a basic search in the project because I was curious
behaviour - 536 times
behavior - 3200 times

Let's not change those, ok?

**Comment by shishurajpandey** [comment]
> Ran a basic search in the project because I was curious behaviour - 536 times behavior - 3200 times
> 
> Let's not change those, ok?

Sure.

**Review by titusfortner — APPROVED** [review]

**Pull request merged**